onadata/apps/main/tests/test_form_auth.py

```diff
@@ -13,11 +14,17 @@ def test_home_redirects(self):
         self._create_user_and_login(username='bob', password='bob')
         response = self.client.get(reverse('home'))
         self.assertEqual(response.status_code, 302)
-        print("response['Location']", response['Location'], flush=True)
+        received, desired = (
+            u.rstrip('/') for u in (response['Location'], settings.KOBOFORM_URL)
+        )
+        assert received == desired
```

Comment:
FYI @noliveleger, I removed `print` statements and added an `assert`, which seems correct and passes for me. LMK if it's wrong